fluent/CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to kubernetes-client
```

Comment:
We should probably delete this file, there's already equivalent on the top level.